LetSwift/Protocols/NibRepresentable.swift

```diff
@@ -21,8 +21,8 @@
 import UIKit
 
 protocol NibPresentable {
-    func registerNib(_ nib: UINib?, forCellReuseIdentifier identifier: String)
-    func registerClass(_ cellClass: AnyClass?, forCellReuseIdentifier identifier: String)
+    func register(_ nib: UINib?, forCellReuseIdentifier identifier: String)
```

Comment:
I changed name of those methods to match updated names of methods in Swift 4

Reply:
Fantastic!